Mimir.Worker/Models/State/AllRuneState.cs

```diff
@@ -0,0 +1,9 @@
+using Libplanet.Crypto;
+using Nekoyume.Model.State;
+
+namespace Mimir.Worker.Models;
+
+public class AllRuneState(Address address, Nekoyume.Model.State.AllRuneState value) : State(address)
+{
+    public Nekoyume.Model.State.AllRuneState Value { get; set; } = value;
```

Comment:
Exists states are named same state name like Inventory, QuestList...
This field name will be database field name, so we need to decided this naming

Reply:
We(w/ @Atralupus) set the convention for this field to `Object`.